test/App/Fossa/FirstPartyScanSpec.hs

```diff
@@ -99,6 +104,26 @@ spec = do
           licenseUnitName secondUnit `shouldBe'` "apache-2.0"
           licenseUnitDataContents unitData `shouldBe'` Nothing
 
+    it' "should find notice files" $ do
+      expectGetOrganizationThatDefaultsToFirstPartyScans
+      licenseSourceUnit <- firstPartyScanWithOrgInfo scanDirWithNoticeFiles Fixtures.standardAnalyzeConfig
+      case licenseSourceUnit of
+        Nothing -> expectationFailure' "first party scan should have run"
+        Just LicenseSourceUnit{licenseSourceUnitLicenseUnits = units} -> do
+          length units `shouldBe'` 3
+          let noticeUnit = NE.head units
+          licenseUnitName noticeUnit `shouldBe'` ""
+          licenseUnitType noticeUnit `shouldBe'` "NoticeFileMatches"
+          let noticeFiles = fromMaybe (NE.fromList []) (licenseUnitNoticeFiles noticeUnit)
+          length noticeFiles `shouldBe'` 1
+          let noticeFile = NE.head noticeFiles
+          let copyrights = fromMaybe (NE.fromList []) (sourceUnitNoticeFileCopyrights noticeFile)
```

Comment:
`fromMaybe (NE.fromList [])` is an error but never gets trigger because of laziness. I think that using `[]` instead of `Maybe NonEmpty ...` makes this a bit more straightforward.